platform.md

```diff
@@ -871,13 +873,15 @@ For more information on build reproducibility see [https://reproducible-builds.o
 ```toml
 [image]
   reference = "<image reference>"
+  mixins = [ "<mixin name>" ]
```

Comment:
```suggestion
  mixins = [ "<mixin name>" ]
  
[run-image]
  reference =  "<image reference>"
  mixins = [ "<mixin name>" ]
```
These are run image mixins and therefore shouldn't be under `[image]` which describes the previous app image. Also we should output the run-image reference, and use that exact reference for the run image in all future steps (removing -run-image and -stack inputs to the `exporter`) to avoid race conditions.

Reply:
Analyzer should either fail or ignore the previous image if the run-image is missing any `mixins` that were present on the previous image.

Reply:
Addressed in 7f95e5fe5fd2b4046ffff631c73d6527d753238c of #172 

I also removed `-run-image` and `-stack` from `exporter` in c173de170916026aed7bde969197e5f4ea3b805a of #172, but I don't think we need a new input to exporter because it already accepts `-analyzed` (we might need to serialize `stack` to `analyzed.toml` though).